app/helpers/pageflow/social_share_helper.rb

```diff
@@ -50,5 +59,9 @@ def social_share_normalize_protocol(url)
       url.gsub(/^(\/\/|https:\/\/)/, 'http://')
     end
 
+    private
+    def strip(text)
+      strip_tags(text.gsub(/<br ?\/?>/, ' ').squish)
+    end
```

Comment:
Private helper methods with short names like this are prone to namespace collisions. If another helper also defines a `strip` method, only one will win and the other will be overwritten.